validator.js

```diff
@@ -546,7 +546,7 @@ function isAlphanumeric(str) {
   throw new Error('Invalid locale \'' + locale + '\'');
 }
 
-var numeric = /^[-+]?[0-9]+$/;
+var numeric = /^[+-]?([0-9]*[.])?[0-9]+$/;
```

Comment:
Hey Amir. The regex looks like it should do the trick but unfortunately it's been implemented directly into the built version of validator.js. You'll want to implement it in the `isNumeric` source file, update the tests, and run the build and commit any changes.

The source file is this one: https://github.com/chriso/validator.js/blob/master/src/lib/isNumeric.js

Reply:
@pietvanzoen , NP
btw what is the difference between the files under `/lib` and the files under `/src/lib` ?

Reply:
@pietvanzoen , done